docs/integrations/bitbucketServer/discovery.md

```diff
@@ -53,6 +55,58 @@ export default async function createPlugin(
 }
 ```
 
+### Installation with Events Support
+
+Please follow the installation instructions at
+
+- <https://github.com/backstage/backstage/tree/master/plugins/events-backend/README.md>
+- <https://github.com/backstage/backstage/tree/master/plugins/events-backend-module-bitbucket-server/README.md>
+
+Additionally, you need to decide how you want to receive events from external sources like
+
+- [via HTTP endpoint](https://github.com/backstage/backstage/tree/master/plugins/events-backend/README.md)
```

Comment:
Might be useful to say that the events webhook url should be sent to:

`${backstageBaseUrl}/api/events/http/bitbucketServer`